examples/rag/callbacks_rag.py

```diff
@@ -38,7 +38,7 @@ def get_checkpoint_callback(output_dir, metric):
         monitor=f"val_{metric}",
         mode="max",
         save_top_k=3,
-        period=0,  # maybe save a checkpoint every time val is run, not just end of epoch.
+        period=1,  # maybe save a checkpoint every time val is run, not just end of epoch.
```

Comment:
why go from 0 to 1?

Reply:
Oh I changed that to speed the the test and forgot to remove it. I can just modify the validation frequency